packages/@n8n/config/src/configs/cache.ts

```diff
@@ -0,0 +1,36 @@
+import { Config, Env, Nested } from '../decorators';
+
+@Config
+class MemoryConfig {
+	/** Max size of memory cache in bytes */
+	@Env('N8N_CACHE_MEMORY_MAX_SIZE')
+	maxSize = 3 * 1024 * 1024; // 3 MiB
+
+	/** Time to live (in milliseconds) for data cached in memory. */
+	@Env('N8N_CACHE_MEMORY_TTL')
+	ttl = 3600 * 1000; // 1 hour
+}
+
+@Config
+class RedisConfig {
+	/** Prefix for cache keys in Redis. */
+	@Env('N8N_CACHE_REDIS_KEY_PREFIX')
+	prefix = 'redis';
```

Comment:
this used to be `cache` https://github.com/n8n-io/n8n/pull/10286/files#diff-ff0f0df9be6611837d92c12ddedac7abf37d0ede7164509c299066a8871837e0L688